react/queries/search.gql

```diff
@@ -49,6 +49,55 @@ query search($query: String, $map: String, $orderBy: String, $priceRange: String
     }
   }
   facets(query: $query, map: $map) @include (if: $withFacets) {
+    # Backwards compatibility with search-result v3
+    Departments {
```

Comment:
I think this could be the last thing, since there's no way to know when this backwards compatibility block ends.

Reply:
ok!